react/components/Modal/modal.css

```diff
@@ -26,3 +18,60 @@
 .scrollBar::-webkit-scrollbar-button {
   display: none;
 }
+
+.contents {
+  display: contents;
+}
+
+.hiddenScroll {
+  overflow: hidden;
+  padding-right: 15px;
+}
+
+.mh60 {
+  max-height: 80vh;
+}
+
+@media screen and (min-width: 40em) {
+  .mw30 {
+    max-width: 30rem;
+  }
+}
+
+@media screen and (min-width: 40em) {
+  .mw40 {
+    max-width: 40rem;
+  }
+}
+
+@media screen and (min-width: 40em) {
+  .mw50 {
+    max-width: 50rem;
+  }
+}
+
+@keyframes fade-in {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
+@keyframes fade-out {
+  0% {
+    opacity: 1;
+  }
+  100% {
+    opacity: 0;
+  }
+}
+
+.openAnimation {
+  animation: fade-in 500ms;
+}
+
+.closeAnimation {
+  animation: fade-out 500ms;
```

Comment:
This animation seems to long, but I don't know

Reply:
It's the same time that had before, I didn't change it